src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -124,13 +133,6 @@ def __init__(
         if coordinates:
             self.region_name = "fill_region"
 
-        if previous_path_to_geojson is not None and coordinates is not None:
```

Comment:
I'm a bit confused, we didn't want to remove this functionality right? Or is that what fill-image.ipynb is for now?

Reply:
sorry fill-image.ipynb was incorrectly commited

Reply:
this if statement should never have been reached, although it could have been used to raise a error.
But for now I will leave it out